deliver/lib/deliver/app_screenshot.rb

```diff
@@ -293,13 +302,27 @@ def self.devices
       }
     end
 
+    def self.resolve_ipadpro_conflict_if_needed(device_type, filename)
+      is_3rd_gen = filename.include?("(3rd generation)")
```

Comment:
`ipadPro129` should also be checked here, that's what downloading the screenshots with deliver will produce for iPad Pro 3rd gen screenshots, then those would also be covered.

Reply:
neat!